From bbcee8849011d702a87cffb03a4c752bdbd7f7e1 Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Fri, 23 Sep 2016 11:30:09 -0700
Subject: [PATCH] IMGAPI-567 Update stud for OpenSSL 1.0.1t/1.0.2h
 vulnerability fixes Reviewed by: Josh Wilsdon <josh@wilsdon.ca> Approved by:
 Trent Mick <trentm@gmail.com>

---
 targets.json.in              | 15 +++++++++++++--
 tools/prep_dataset_in_jpc.sh |  6 +++---
 2 files changed, 16 insertions(+), 5 deletions(-)

diff --git a/targets.json.in b/targets.json.in
index 6ad7448..5a579e9 100644
--- a/targets.json.in
+++ b/targets.json.in
@@ -584,9 +584,20 @@ cat <<EOF
     "appliance": "true",
     "image_name": "imgapi",
     "image_description": "SDC IMGAPI",
-    "image_uuid": "fd2cc906-8938-11e3-beab-4359c665ac99",
+    "// image_uuid": "sdc-minimal-multiarch-lts@15.4.1",
+    "image_uuid": "18b094b0-eb01-11e5-80c1-175dac7ddf02",
     "pkgsrc": [
-        "xz-5.0.3"
+        "coreutils-8.23nb2",
+        "curl-7.47.1",
+        "dateutils-0.3.1nb1",
+        "grep-2.22",
+        "gsed-4.2.2nb4",
+        "patch-2.7.5",
+        "xz-5.2.2",
+        "haproxy-1.6.2",
+        "stud-0.3p53nb5",
+        "sudo-1.8.15",
+        "the_silver_searcher-0.31.0"
     ],
     "repos": [
       {"url": "git@github.com:joyent/sdc-imgapi.git"}
diff --git a/tools/prep_dataset_in_jpc.sh b/tools/prep_dataset_in_jpc.sh
index 6d2c5d9..c8252bc 100755
--- a/tools/prep_dataset_in_jpc.sh
+++ b/tools/prep_dataset_in_jpc.sh
@@ -407,9 +407,9 @@ else
 fi
 
 # Image Notes:
-# - We need to add a requirement on the manifest for networks but CloudAPI does
-#   not allow us to do that, so we have to change locally and push over the
-#   original.
+# - requirements.networks: Feels right to have the images saying they need
+#   to be on the admin network. (However, it isn't required for either of
+#   headnode setup or 'sdcadm up'.)
 # - We also set the min_platform to the platform we just built the bits on,
 #   not the platform we created the image on, since that's where the binary
 #   dependency should come from.
-- 
2.21.0

